src/Scripting/Core/ScriptOptions.cs

```diff
@@ -54,6 +53,7 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Reflection.Extensions",
                 "System.Reflection.Primitives",
                 "System.Runtime",
+                "System.Runtime.Extensions",
```

Comment:
My mistake, it's actually not `dynamic`, it's `Environment`. It looks like two scripting tests are failing without it: `CompilationChain_UsingDuplicates` and `CompilationChain_GlobalImports

---
In reply to: [110970347](https://github.com/dotnet/roslyn/pull/18471#discussion_r110970347) [](ancestors = 110970347)